packages/components/src/components/step-group/step-group.stories.ts

```diff
@@ -0,0 +1,139 @@
+        </sd-step>
+
+        <sd-step size="lg" orientation="horizontal" state="inProgress">
+          <p slot="label">Exercitation ullamco laboris</p>
+          Lorem ipsum est dolor sit amet
+        </sd-step>
+
+        <sd-step size="lg" orientation="horizontal" state="waiting">
+          <p slot="label">Reprehenderit qui in e name</p>
+          Lorem ipsum est dolor sit amet
+        </sd-step>`
+    }
+  ]),
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+
+/**
+ * Default: This shows sd-step-group in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return html`${generateTemplate({ args })}`;
+  }
+};
+
+/**
+ * Use the 'base' and 'body' parts to style the step-group.
+ */
+export const Parts = {
+  parameters: {
+    controls: {
+      exclude: ['base', 'body']
+    }
+  },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        x: { type: 'attribute', name: 'orientation' },
+        y: {
+          type: 'template',
+          name: 'sd-step-group::part(...){outline: solid 2px red}',
+          values: ['base', 'body'].map(part => {
+            return {
+              title: part,
+              value: `<style>#part-${part} sd-step-group::part(${part}){outline: solid 2px red; outline-offset: -2px;}</style><div id="part-${part}">%TEMPLATE%</div>`
+            };
+          })
+        }
+      },
+      args
+    });
+  }
+};
+
+/**
+ * sd-steps are fully accessibile via keyboard.
+ */
+
+export const Mouseless = {
+  render: (args: any) => {
+    return html`<div class="mouseless">
+      ${generateTemplate({
+        args
+      })}
+    </div>`;
+  },
+
+  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
+    const el = canvasElement.querySelector('.mouseless sd-step-group');
+    await waitUntil(() => el?.querySelector('sd-step')?.shadowRoot?.querySelector('button'));
+
+    el?.querySelector('sd-step')?.shadowRoot?.querySelector('button')!.focus();
+  }
+};
+
+/**
+ * This sample shows how to set the active step programmatically.
+ */
+
+export const SetActiveStep = {
+  name: 'Sample: Set Active Step',
+  render: () => {
+    return html`
+      <sd-step-group id="set-active" size="lg" orientation="horizontal" activestep="0">
+        <sd-step size="lg" orientation="horizontal" state="finished">
+          <p slot="label">Lorem ipsum dolor sit</p>
+          Lorem ipsum est dolor sit amet
+        </sd-step>
+
+        <sd-step size="lg" orientation="horizontal" state="inProgress">
+          <p slot="label">Exercitation ullamco laboris</p>
+          Lorem ipsum est dolor sit amet
+        </sd-step>
+
+        <sd-step size="lg" orientation="horizontal" state="waiting">
+          <p slot="label">Reprehenderit qui in e name</p>
+          Lorem ipsum est dolor sit amet
+        </sd-step>
+      </sd-step-group>
+
+      <sd-button class="w-min mt-8" size="sm" id="next">Next</sd-button>
```

Comment:
Please make a `back` button, too. The UX isn't fun that you have to up to go to a step back.

Reply:
Would not agree.
The idea here was to show that people can click on the completed step to directly jump back to it. A back button would make the whole story obsolete IMO.

maybe we can refactor the description here to make clear what we want to show?!

Reply:
I think it could be useful to include the back button since the story is mainly to show that you can programatically change the steps. 

Instead I could add the information about clicking to the description (I think a description alone could be enough as we now have hover styles on finished steps that indicate "click-ability" anyways)

Reply:
in general, if we talk about UX, we could include our designers into this... ;-)

In specific, I like the idea to only have the description. It makes it even more clear, that we don't talk about a functionality we provide but one we want to show...